src/OrchardCore.Modules/OrchardCore.Lucene/Deployment/LuceneIndexRebuildDeploymentStepDriver.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OrchardCore.Deployment;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Lucene.ViewModels;
+
+namespace OrchardCore.Lucene.Deployment
+{
+    public class LuceneIndexRebuildDeploymentStepDriver : DisplayDriver<DeploymentStep, LuceneIndexRebuildDeploymentStep>
+    {
+        private readonly LuceneIndexSettingsService _luceneIndexSettingsService;
+
+        public LuceneIndexRebuildDeploymentStepDriver(LuceneIndexSettingsService luceneIndexSettingsService)
+        {
+            _luceneIndexSettingsService = luceneIndexSettingsService;
+        }
+
+        public override IDisplayResult Display(LuceneIndexRebuildDeploymentStep step)
+        {
+            return
+                Combine(
+                    View("LuceneIndexRebuildDeploymentStep_Fields_Summary", step).Location("Summary", "Content"),
+                    View("LuceneIndexRebuildDeploymentStep_Fields_Thumbnail", step).Location("Thumbnail", "Content")
+                );
+        }
+
+        public override IDisplayResult Edit(LuceneIndexRebuildDeploymentStep step)
+        {
+            return Initialize<LuceneIndexRebuildDeploymentStepViewModel>("LuceneIndexRebuildDeploymentStep_Fields_Edit", async model =>
+            {
+                model.IncludeAll = step.IncludeAll;
```

Comment:
Shouldn't this be a separate config instead of listing all current indices? Because then "include all" will continue to work even if new indices are added.

Reply:
Now it is consistent with other steps even with those from Elasticsearch PR.

Reply:
That's an issue as well, then. My above remark still stands.

Reply:
fixed